README.md

```diff
@@ -1 +1,10 @@
-# Hardware_Rental_Store
\ No newline at end of file
+# Hardware_Rental_Store
+### Project3 for CSCI5448 (OOAD)
+
+Team Member:
+* Yutaka Urakami
+ * Mengyu Wu
+ * Hao Wu
+ 
```

Comment:
Please write the location of the pdf like output.txt.
And, add the pdf in the repository